projects/plugins/jetpack/extensions/blocks/ai-paragraph/edit.js

```diff
@@ -88,7 +146,7 @@ export default function Edit( { attributes, setAttributes } ) {
 				setErrorMessage
 			);
 		}
-	}, [ attributes, formattedPrompt, setAttributes ] );
+	}, [] );
```

Comment:
The below linter warning is wrong

When I pass this, this will auto-update when people are clicking in the interface and I DO NOT want that.